rules/legacy_xcodeproj.bzl

```diff
@@ -860,7 +860,6 @@ def _populate_xcodeproj_targets_and_schemes(ctx, targets, src_dot_dots, all_tran
         target_settings["BAZEL_LLDB_INIT_FILE"] = lldbinit_file
 
         if product_type == "application" or product_type == "app-extension":
-            target_settings["INFOPLIST_FILE"] = "$BAZEL_STUBS_DIR/Info-stub.plist"
```

Comment:
Does this cause any issues on Xcode 13 / legacy build system?

Reply:
Ok as long as you tested it out. Perhaps it was required to have this set in vintage releases but not on modern cuts